owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -237,4 +237,8 @@ public static boolean deleteDir(File dir) {
 
         return dir.delete();
     }
+
+    public static String getDefaultCameraSourcePath() {
+        return Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DCIM).getAbsolutePath() + "/Camera";
```

Comment:
Let's move `/Camera` to a constant. We will try to remove the hardcoded strings.